cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -399,6 +399,9 @@ public synchronized UserTaskInfo markTaskInExecution(String uuid) {
 
     if (_uuidToActiveUserTaskInfoMap.containsKey(userTaskId)) {
       _inExecutionUserTaskInfo = _uuidToActiveUserTaskInfoMap.remove(userTaskId).setState(TaskState.IN_EXECUTION);
+      // Normally a user task's operation result is logged when the task's state is transferred from ACTIVE to COMPLETED_WITH_ERROR.
+      // If the user task's state is transferred from ACTIVE directly to IN_EXECUTION, need to log the task's operation result here.
```

Comment:
Can we draw the full state machine under the JavaDoc of `TaskState`? Currently it is not clear which state is supposed to go to which final state.